services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankealueEntity.kt

```diff
@@ -30,7 +30,7 @@ class HankealueEntity : HasId<Int> {
     var meluHaitta: Haitta13? = null
     var polyHaitta: Haitta13? = null
     var tarinaHaitta: Haitta13? = null
-    var nimi: String? = null
+    var nimi: String = "$HANKEALUE_DEFAULT_NAME 1"
```

Comment:
We wouldn't need a default value here, if this variable was defined in the constructor. Preferably along with the other properties Like:
```
class HankealueEntity(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY) override var id: Int = 0,
    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "hankeid")
    var hanke: HankeEntity? = null,

    // NOTE: This should be changed to an entity
    // Refers to HankeGeometria.
    var geometriat: Int? = null,
    var haittaAlkuPvm: LocalDate? = null,
    var haittaLoppuPvm: LocalDate? = null,
    var kaistaHaitta: TodennakoinenHaittaPaaAjoRatojenKaistajarjestelyihin? = null,
    var kaistaPituusHaitta: KaistajarjestelynPituus? = null,
    var meluHaitta: Haitta13? = null,
    var polyHaitta: Haitta13? = null,
    var tarinaHaitta: Haitta13? = null,
    var nimi: String,
) : HasId<Int> {
```

Reply:
Changed to suggested